Dockerfile

```diff
@@ -5,12 +5,15 @@ RUN /usr/src/ckan-cloud-operator/docker-build.sh
 COPY environment.yaml /environment.yaml
 RUN conda env create -f /environment.yaml
 RUN echo conda activate ckan-cloud-operator >> ~/.bashrc &&\
+    echo 'if [ -z "$NO_KUBE_CONFIG" ]; then' >> ~/.bashrc &&\
     echo '[ -e /etc/ckan-cloud/.kube-config ] && export KUBECONFIG=/etc/ckan-cloud/.kube-config' >> ~/.bashrc &&\
     echo '! [ -z "${KUBE_CONTEXT}" ] && kubectl config use-context "${KUBE_CONTEXT}" >/dev/null 2>&1' >> ~/.bashrc &&\
     echo '! [ "$(ckan-cloud-operator config get --key=ckan-cloud-provider-cluster-main-provider-id --raw)" == "aws" ] && ckan-cloud-operator activate-gcloud-auth >/dev/null 2>&1' >> ~/.bashrc &&\
     echo 'ckan-cloud-operator db proxy port-forward --all-daemon "I know the risks" >/dev/null 2>&1' >> ~/.bashrc &&\
-    echo 'while ! pg_isready -h localhost >/dev/null 2>&1; do sleep .1; done' >> ~/.bashrc
+    echo 'while ! pg_isready -h localhost >/dev/null 2>&1; do sleep .1; done' >> ~/.bashrc &&\
+    echo 'fi' >> ~/.bashrc
```

Comment:
These echo's seem kind of error prone. I find that it's easy to mess up the shell quoting or the shell redirection. Why not just put these in a separate file, and then append it? e.g.
```
RUN cat bashrc.inc >> ~/.bashrc
```

Reply:
I agree